povertymapping/hdx.py

```diff
@@ -210,4 +209,32 @@ def get_hdx_file(
 
     logger.info(f"HDX Data: Successfully downloaded and cached for {region} at {zipfile_path}!")
 
-    return unzipped_hdxfile
\ No newline at end of file
+    return unzipped_hdxfile
+
+
+def generate_hdx_features(
```

Comment:
Might be be clearer if the `hdx` utils were renamed to `hrsl` instead, since `hdx` is a huge repository containing many other types of datasets apart from hrsl population. Here we're only considering HRSL population

Reply:
Makes sense! To clarify which `hdx` utils are you referring to that can be renamed?
- the `hdx.py` filename
- the `generate_hdx_features()` function
- all function and variable names in `hdx.py` that have `hdx` but actually mean `hrsl`

Reply:
I was thinking renaming all `hdx` to `hrsl` since `hdx` isn't really a dataset but more of a general data hosting service

Do you have in mind specific functions/names/variables can be better kept as `hdx`?

Reply:
I recommend to retain these as `hdx` since they're the config for calling the HDX Python API:
- `HDX_CONFIG`
- `init_hdx_config()`

All other cases, seems more appropriate to be `hrsl`:
- `hdx.py`
- `download_hdx()`
- `get_unzipped_hdxfile()`
- `get_hdx_file()`

Reply:
oh there are utilities using the HDX python API, oops I missed that. Thanks!